tests/pytorch_tests/model_tests/feature_models/add_net_assert_test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Sony Semiconductors Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import torch
+from torch import _assert
+from tests.pytorch_tests.model_tests.base_pytorch_test import BasePytorchTest
+
+"""
+This tests checks that the assert operation is being removed from the graph during quantization.
+"""
+
+
+class AddAssertNet(torch.nn.Module):
```

Comment:
Please change the class name to "AssertNet", and the file name to "remove_assert_test.py"